### PR TITLE
calling window for performance now

### PR DESCRIPTION
**Pull request by alanpoon**
Fix to #4

### CODE REVIEW

**Comment by sebcrozet** [comment]
Hi @alanpoon!

Would you like to describe the goal of this PR?
A few remarks:

- The field of `wasm::Instant` should not be public because it would break parity with `Instant` from the std lib.
- The `use wasm_bindgen` statements should be directly inside the `fn now` function. That way they will be affected by the `#[cfg(feature = "wasm-bindgen")]`
- The `web-sys` dependency should be optional, and activated only when we are targeting WASM with `wasm-bindgen`.

File: Cargo.toml [hunk 1]
```diff
@@ -32,3 +32,6 @@ wasm-bindgen = { version = "0.2", optional = true }
 
 [dev-dependencies]
 wasm-bindgen-test = "0.2"
+
+[target.'cfg(feature="wasm-bindgen")'.dependencies]
+web-sys = {version="0.3.19", optional = true, features=['Window', 'Performance', 'PerformanceTiming']}
```
Comment (hecrj) [inline Cargo.toml]: I do not think we can check enabled features in a `target.cfg(...)`. I think we can only check target-related attributes: https://doc.rust-lang.org/cargo/reference/config.html

Comment (hecrj) [reply Cargo.toml]: We probably want something like this:
```toml
[features]
now = ["time"]
wasm-bindgen = ["wasm_bindgen", "web_sys"]

[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
time = { version = "0.1", optional = true }

[target.'cfg(target_arch = "wasm32")'.dependencies]
stdweb = { version = "0.4", optional = true }
wasm_bindgen = { package = "wasm-bindgen", version = "0.2", optional = true }
web_sys = { package = "web-sys", version = "0.3", optional = true }
```

**Comment by sebcrozet** [comment]
Closing this in favor of #6.

**Pull request closed without merging**